js/src/forum/components/CreatePollModal.js

```diff
@@ -109,8 +112,9 @@ export default class CreatePollModal extends Modal {
     }
 
     displayOptions() {
+        console.log(this.options);
         return Object.keys(this.options).map((el, i) => (
-            <div className={this.options[i + 1] === '' ? 'Form-group hide' : 'Form-group'}>
+            <div className={classList('Form-group', this.options[i + 1]() === '' && 'hide')}>
```

Comment:
console error here

![image](https://user-images.githubusercontent.com/16573496/119638566-b3c84c00-be0e-11eb-95db-7fcd54c1f75f.png)

Reply:
I had that error while developing then I thought that I resolved it but I guess my changes got lost into the void...
Now I tried to understand why this is there anyway and found out that this case actually never exists (` this.options[i + 1]() === ''` so I just removed it. Also in EditPollModal.js it's not present so I guess that it isn't needed. 

Maybe @datitisev knows better here ?